### PR TITLE
[docs] Update OpenBAS x OpenCTI integration

### PR DESCRIPTION
**Pull request by RomuDeuxfois**

### CODE REVIEW

File: docs/usage/scenario/opencti_scenario.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+- Reports
+- Grouping
+- Incident Response
+- Request For Information
+- Request For Takedown
+
+![simulate button](assets/simulate-btn.png)
+
+When you click the simulate button, you’ll have two options:
```
Comment (EllynBsc) [inline docs/usage/scenario/opencti_scenario.md:17]:
```suggestion
When you click on the simulate button, you’ll have two options:
```

File: docs/usage/scenario/opencti_scenario.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+- Generate a scenario based on technical injects
+- Generate a scenario based on email injects, using AI to automatically generate email content
+
+It’s essential to understand that scenario creation for these entities relies on matching TTPs between OpenCTI and
```
Comment (EllynBsc) [inline docs/usage/scenario/opencti_scenario.md:22]:
```suggestion
It’s essential to understand that a scenario creation for these entities relies on matching TTPs between OpenCTI and
```

File: docs/usage/scenario/opencti_scenario.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+When generating a scenario from OpenCTI, a scenario is created and can be accessed from the scenarios screen. The
+scenario name will include a reference to OpenCTI, indicating its origin. This scenario will automatically contain
+relevant sequences of injects based on the threat context identified in OpenCTI.
+
+However, it's important to review and potentially customize the scenario to ensure it meets your organization's specific
+requirements. Additionally, you'll need to select appropriate [targets](../targets.md) for the injects within the
+scenario.
+
+Once you've finalized the scenario, you can schedule your simulation as you would for any other scenario. The overall
```
Comment (EllynBsc) [inline docs/usage/scenario/opencti_scenario.md:34]:
```suggestion
Once you've finalized the scenario, you can schedule your simulation as you would do for any other scenario. The overall
```

**Review by EllynBsc — CHANGES_REQUESTED** [review]
Hey @RomuDeuxfois 👋 

Really nice!

**Comment by RomuDeuxfois** [comment]
@EllynBsc  I added all your changes :)
Thanks for the review !

**Review by EllynBsc — APPROVED** [review]

**Pull request merged**